src/MenuItem/index.tsx

```diff
@@ -64,7 +65,13 @@ interface Props
    * This element will always be rendered unless the value is `undefined`. If
    * you want an empty node, a spacer for example, use `null`
    */
-  icon?: React.ReactNode;
+  leftIcon?: React.ReactNode;
```

Comment:
nbd: we have a precedent for `iconLeft` and `iconRight` in parts of FE codebase, maybe consistency would be preferred.

Reply:
I can't put my finger on _why_, but I prefer it to sound conversational, left icon versus icon on the left. Also, after looking at the material ui naming conventions, I like the idea of copying flexbox's naming conventions and using `startIcon` and `endIcon`.

Reply:
Renamed.